crates/fj-operations/src/lib.rs

```diff
@@ -51,16 +51,16 @@ pub trait Shape {
 }
 
 impl Shape for fj::Shape {
-    type Brep = FaceSet;
+    type Brep = Set<Face>;
 
     fn compute_brep(
         &self,
         services: &mut Services,
         debug_info: &mut DebugInfo,
     ) -> Self::Brep {
         match self {
-            Self::Shape2d(shape) => {
-                shape.compute_brep(services, debug_info).faces().clone()
+            Self::Shape2d(_shape) => {
+                todo!()
```

Comment:
Now that sketch is completely 2D, the `Brep` of a `Shape2d` can't be a `FaceSet`

Reply:
Allowing the `Brep` to be a sketch, we would run into a problem where in:
 https://github.com/hannobraun/Fornjot/blob/a2579f6ee99793a9bdc12b499454dd8d9667fec1/crates/fj-operations/src/transform.rs#L10-L20
We need to be able to transform the `Brep` which we can't do to a `Sketch`.
I think for now I'll implement it using the xy-plane :+1: